lib/suma/mobility.rb

```diff
@@ -20,4 +20,25 @@ def self.int2coord(i)
     raise OutOfBounds, "#{i} must be between -1.8b and 1.8b" unless INTCOORD_RANGE.cover?(i)
     return i * INT2COORD_FACTOR
   end
+
+  # Same location vehicles are spread equally around that location in a circular manner.
+  # We pass an offset variable for each vehicle to evaluate their new position.
```

Comment:
Explain what `map_vehicles` consists of, in terms of keys. We normally don't pass around formless hashes between API and library methods since it is very difficult to keep track of types.